src/components/form/version/VersionFields.js

```diff
@@ -12,7 +12,7 @@ import DateTimePicker from "@/components/date-time/DateTimePicker";
 
 
 export default function VersionFields(props) {
-    const [qualityDesingation, setQualityDesingation] = useState("");
+    const [qualityDesingation, setQualityDesingation] = useState(props.fieldValues?.quality_designation || "");
```

Comment:
```suggestion
    const [qualityDesignation, setQualityDesignation] = useState(props.fieldValues?.quality_designation || "");
```
Same typo in other files, can search the repo for them

Reply:
Good spot. Think it fixed all the files but feel free to check :)